include/envoy/server/configuration.h

```diff
@@ -128,10 +128,31 @@ class Main {
   virtual Optional<uint32_t> statsdUdpPort() PURE;
 
   /**
-   * @return the time interval between flushing to configured stat sinks. The server latches
-   *         counters.
+   * @return std::chrono::milliseconds the time interval between flushing to configured stat sinks.
+   *         The server latches counters.
    */
   virtual std::chrono::milliseconds statsFlushInterval() PURE;
+
+  /**
+   * @return std::chrono::milliseconds the time interval after which we count a nonresponsive thread
+   *         event as a "miss" statistic.
+   */
+  virtual std::chrono::milliseconds wdMissTimeout() const PURE;
+  /**
```

Comment:
nit: newlines between functions defs and next comment to match style elsewhere (same below)